app/support/reservations/relay_support.rb

```diff
@@ -32,8 +32,9 @@ def can_kill_power?
 
   def other_reservations_using_relay
     order_detail.reservation.product.schedule.reservations
-      .active_without_admin
+      .active_with_admin
```

Comment:
Do we actually now need the `active_with_admin` and `active_without_admin` methods anymore? Can we go back to just `active` like we had before?